code/datums/components/examine_balloon.dm

```diff
@@ -61,10 +62,12 @@
 
 	var/mutable_appearance/examine_arrow = mutable_appearance('icons/effects/effects.dmi', "examine_arrow")
 	examine_arrow.pixel_y = pixel_y_offset_arrow - balloon.pixel_y
+	var/matrix/new_transform = matrix()
+	new_transform.Scale(1 / size_upscaling, 1 / size_upscaling)
+	examine_arrow.transform = new_transform
```

Comment:
Why are you shrinking the arrow?

Reply:
because its an overlay on the object we're scaling up